src/tests/EdFi.DataManagementService.Tests.E2E/Features/General/DiscoveryAPI.feature.disabled

```diff
@@ -0,0 +1,73 @@
+Feature: The Discovery API provides information about the application version, supported data model(s), and URLs for additional metadata.
+
+        @ignore
+        Scenario: 01 GET / returns the root Discovery API document
+            Given a GET to the root of the API
+             Then it should respond with 200
+              And the response body is
+                  """
+                  {
+                      "version": "1.0.0",
+                      "applicationName": "Ed-Fi Alliance Data Management Service"
+                      "dataModels": [
+                          {
+                              "name": "Ed-Fi",
+                              "version": "5.1.0",
+                              "informationalVersion": "The Ed-Fi Data Standard v5.1"
+                          }
+                      ],
+                      "urls": [
+                          "dependencies": "{BASE_URL}/metadata/dependencies",
+                          "openApiMetadata": "{BASE_URL}/metadata/specifications",
+                          "oauth": "{BASE_URL}/oauth/token",
+                          "dataManagementApi": "{BASE_URL}/data",
+                          "xsdMetadata": "{BASE_URL}/metadata/xsd",
+                          "changeQueries": "{BASE_URL}/changeQueries/v1",
+                          "identity": "{BASE_URL}/identity/v2"
```

Comment:
Sorry that i missed these two: they are optional components that the DMS won't support yet.

Reply:
Items removed